src/runtime_src/hip/core/event.cpp

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+// Copyright (C) 2024 Advanced Micro Device, Inc. All rights reserved.
```

Comment:
Undo this change please.  It is `Devices` not `Device`

Reply:
fixed.